.eslintignore

```diff
@@ -0,0 +1 @@
+app/javascript/packs/jest
```

Comment:
Why not ignore all the packs? and node_modules?

Reply:
Right now the npm script ( package.json ) just runs eslint against `app/javascript/pack` but we want to ignore the jest config.

Reply:
👍